server/database/sql/create_triggers.sql

```diff
@@ -9,3 +9,33 @@ before
 update on tracker
 for each row
 execute procedure trigger_set_timestamp();
+
+CREATE TRIGGER insert_words_from_overview AFTER INSERT 
+ON tracker
+FOR EACH ROW
+execute procedure trigger_insert_words()
+
+CREATE TRIGGER delete_words_from_overview AFTER UPDATE 
+ON tracker
+FOR EACH ROW
+execute procedure trigger_delete_words()
+
+CREATE TRIGGER update_words_from_overview AFTER UPDATE 
+ON tracker
+FOR EACH ROW
+execute procedure trigger_insert_words()
```

Comment:
@ChrisShep98 these two get triggered after an update is run on the tracker table. when they're triggered, we both delete all the words in the word table from todays submission, and insert all the words from the updated submission